app/javascript/components/books.vue

```diff
@@ -6,9 +6,13 @@
     <template v-else>
       <div class="description">
         <span class="emoji">💁🏻‍♀️💁🏽‍♂️</span>
-        The library has <strong>{{ this.book_count }}</strong> of your recently added to-read books:
+        The library has <strong>{{ this.book_count }}</strong> of your recently added to-read books.
+        <div class="sort-controls">
+          <button @click="this.sortByRatingDesc">Highest rated first</button>
+          <button @click="this.sortByYearDesc">Most recently published first</button>
+        </div>
       </div>
-      <book :bookObj="book" v-for="book in books" />
+      <book :bookObj="book" v-for="book in books" :key="book.title" />
```

Comment:
Will update `library_assistant` to include the ISBN later so we have something nicer to use as the key.